src/style.css

```diff
@@ -2,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. You should add only top or bottom margins but not both to avoid potential margin collapse issues. In this case, you're adding a right margin, which is fine, but just be mindful of top and bottom margins.